File: java/Raytracer with shadows and IO/Src/PointLight.java
public class PointLight
{
  public Vector3 location = new Vector3( );
  public RGBColor color = new RGBColor( );
  
  public Vector3 getLightVector(Vector3 point)
  {
    return Vector3Util.normalize(Vector3Util.difference(location, point));
  }
}

File: src/Tracer/PointLight.cs
using System;
using System.Numerics;

namespace Tracer
{    
    public class PointLight : Light
    {
        public Vector3 Location { get; set; }
        public override Vector3 GetLightVector(Vector3 point)
        {
            if (point == null)
            {
                throw new ArgumentNullException(nameof(point));
            }
            
            return Vector3.Normalize(Location - point);
        }
    }
}